### PR TITLE
Send packages from Connect to CAPI in CreateLabel response

### PR DESCRIPTION
**Pull request by mikehw**
Previously we were not sending package-level tracking numbers and identifiers to the Carrier API in the create label response.

With this change, we will now be mapping that information from the connect createShipment.

### CODE REVIEW

File: packages/shipping-runtime/src/mapping/functions/create-label-response.ts [hunk 1]
```diff
@@ -31,6 +32,7 @@ export const mapCreateLabelResponse = (
     insurance_amount: insuranceCost,
     estimated_delivery_datetime: mapDateTime(response.deliveryDateTime),
     carrier_transaction_id: response.identifiers?.carrierTransactionId,
+    packages: getLabelPackages(response.packages),
```
Comment (justin-shipstation) [inline packages/shipping-runtime/src/mapping/functions/create-label-response.ts:35]: I think you could probably just call response.packages?.map(mapLabelPackage);

File: packages/shipping-runtime/src/mapping/functions/get-label-packages.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import { Identifiers } from "@shipengine/connect-sdk";
+import { PackageConfirmation } from "@shipengine/connect-sdk";
+
+export function getLabelPackages(
+  packageConfirmations: readonly PackageConfirmation[] | undefined
+): LabelPackage[] | undefined {
+  return packageConfirmations?.map(getLabelPackage);
+}
+
+function getLabelPackage(
```
Comment (justin-shipstation) [inline packages/shipping-runtime/src/mapping/functions/get-label-packages.ts:14]: export const mapLabelPackage = () => {}

File: packages/shipping-runtime/src/mapping/functions/get-label-packages.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+): LabelPackage {
+  return {
+    tracking_number: packageConfirmation.trackingNumber,
+    alternative_identifiers: getAlternativeIdentifiers(
+      packageConfirmation.identifiers
+    ),
+  };
+}
+
+function getAlternativeIdentifiers(
```
Comment (justin-shipstation) [inline packages/shipping-runtime/src/mapping/functions/get-label-packages.ts:25]: To keep things cohesive, lets use function expression here `const getAlternativeIdentifiers = () => {}`

**Review by justin-shipstation — COMMENTED** [review]
A few update requests.

**Review by justin-shipstation — APPROVED** [review]

**Comment by mikehw** [comment]
release carrier-api,shipping-runtime,connect minor

**Pull request merged**